test/tools/verify/verify.go

```diff
@@ -0,0 +1,289 @@
+package verify
```

Comment:
we could make it part of `e2e / provisioning` pkg

Reply:
Yeah good idea, I wasn't quite sure if we maybe still need the tool